requirements.txt

```diff
@@ -1 +1 @@
-ansible>=2.2.0
+ansible<2.3
```

Comment:
FWIW, I think 2.3 is what's currently shipping in CentOS.  The little bits of tinkering I've done with it in ADEPT have been mostly positive.  Though IIRC it does alter some behaviors in non-backward compatible ways, the fixes needed were pretty minor.